systemvm/patches/debian/vpn/etc/ipsec.d/l2tp.conf

```diff
@@ -30,4 +35,5 @@ conn L2TP-PSK
         # ----------------------------------------------------------
         # Change 'ignore' to 'add' to enable this configuration.
         #
+        rightsubnetwithin=10.1.2.0/8
```

Comment:
Default is 10.1.2.0/8. In configure.py 'configure_l2tpIpsec' method will update its value.

 We have only ip range ex:10.1.2.2-10.1.2.8 we don't have the cidr in DB. In global setting we have option to configure only range not the cidr. We have to derive the cidr from the range and it may not be correct some times.  
If we hard code to /24  it may will be problem if the range crossing the /24. 
If we introduce the global setting to configure the cidr, it is mess for the backward compatibility.